packages/cli/src/update.js

```diff
@@ -69,6 +69,11 @@ export async function checkForUpdate() {
   let pkg = getPackageJSON(import.meta.url);
   let log = logger('cli:update');
 
+  if (process.env.PERCY_SKIP_UPDATE_CHECK === 'true') {
+    log.debug('Skipping update check');
+    return;
+  }
```

Comment:
Shouldn't we allow any truthy value for this?

Reply:
hmm, makes sense. so just `if(process.env.PERCY_SKIP_UPDATE_CHECK){ ... }` will work